src/commands/Music/add.ts

```diff
@@ -13,6 +13,7 @@ export default class extends MusicCommand {
 	}
 
 	public run(message: KlasaMessage, [songs]: [Track | Track[]]) {
+		if (!songs) throw null;
```

Comment:
If this ever happens, throw a more descriptive error, not just `null`, you can break CommandError easily lol